server/config.go

```diff
@@ -68,8 +68,9 @@ type Config struct {
 
 	BalanceCfg BalanceConfig `toml:"balance" json:"balance"`
 
-	// Only test can change it.
-	nextRetryDelay time.Duration
+	// Only test can change them.
+	nextRetryDelay      time.Duration
+	strictReconfigCheck string
```

Comment:
Because I only want use `false` in tests, but `false` happens to be the default value of a boolean, so I need a third status to distinguish these two cases.

Please review the following change:
https://github.com/pingcap/pd/pull/302/commits/0a66163f346917488a18c2d31a5cfc08a515b16e#diff-a8a8090b66912f265b621fd23193869cR411
https://github.com/pingcap/pd/pull/302/commits/0a66163f346917488a18c2d31a5cfc08a515b16e#diff-a8a8090b66912f265b621fd23193869cR467